lib/plugins/rollback/tests/rollback.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Rollback = require('../index');
+const Serverless = require('../../../Serverless');
+
+describe('Rollback', () => {
+  let rollback;
+  let serverless;
+
+  beforeEach(() => {
+    serverless = new Serverless();
+    rollback = new Rollback(serverless);
+  });
+
+  describe('#constructor()', () => {
+    it('should have the command rollback', () => {
+      // eslint-disable-next-line no-unused-expressions
```

Comment:
Yep. I faced that issue a couple of times and always rewrote the test in a way that the linter was happy. Thanks for the explanation @nikgraf